src/content/blog/2024-plans.md

```diff
@@ -0,0 +1,47 @@
+
+## Sustainability and Finance
+
+* **Continue consistent marketing**, including blogs, clear fundraising efforts, newsletter. In 2023 we made marketing outreach a regular goal, as it had been neglected in the past. We want everybody to know that DDEV's future sustainability depends on them, and want them to be in the loop on what's going on. As a result of this, we now have a monthly [newsletter](/newsletter) and better communication around funding and supporting the project. We still have a long ways to go on this, and as always it competes for attention and time with other priorities.
+* **Recruit, train, and add a third paid co-maintainer**: Our two maintainers are 100% occupied with current priorities, and we should be able to grow funding (and maintainer capabilities) to add another co-maintainer.
+
+## Features
+
+* **Explicit [Upsun](https://upsun.com/)** by Platform.sh support, probably meaning an add-on for it, at least supporting the documented target CMSs.
+* **Improve xhprof usage**. The [ddev-xhgui](https://github.com/tyler36/ddev-xhgui) add-on has demonstrated how much better the xhprof experience can be in DDEV. We would like to incorporate the ddev-xhgui work into DDEV core and make the great experience available to everyone without an add-on.
+* **Refactor configuration code** to promote consistent handling of config (global and project) and generated docs. Currently, there is a lot of technical debt in the handling of project and global configuration, with code often repeated for each setup. This can be reworked and give consistent approach to these, while removing much code. This could include using Viper for configuration, which would allow environmental overrides, etc.
+* **Node.js back-end**: DDEV already has lots of support for Node.js in many contexts, but it would be great to have the ability to replace php-fpm with a node server, and this is already not hard to do, but would be great as a DDEV feature.
+* **Add-on Registry**: The available DDEV Add-ons have expanded enormously since the launch of this feature. We probably need something fancier then `ddev get --list` to show and explain them. This can be API-based and should work fine.
+
+## Architecture
+
+* **Discuss breaking up `ddev-webserver`** into multiple images/containers
+  * The base image could be smaller
+  * PHP could be a separate image/container
+  * Apache and Nginx versions could be separate images
+* **Add swapability of the web server component**: Currently we have Apache and Nginx built-in, but people periodically ask for Caddy, etc.
+* **Allow other back-ends (like Node.js and frankenphp)** in addition to current php-fpm and gunicorn. Make them swappable so that future back-ends could easily be accommodated.
```

Comment:
```suggestion
* **Support more back-ends (like Node.js and frankenphp)** in addition to the current php-fpm and gunicorn. Make them swappable so that future back-ends could easily be accommodated.
```